Application/EdFi.Ods.Tests/EdFi.Ods.Security/Authorization/EdFiAuthorizationProviderTests.cs

```diff
@@ -601,7 +601,7 @@ protected override void Act()
                     // rather than the order in which the claims are issued to the caller in the claim set.
                     // In this case, Resource2 is "lower" in the hierarchy, and should be the one matched first.
                     JsonClaimHelper.CreateClaim(Resource3ClaimUri, new EdFiResourceClaimValue(ReadActionUri)),
-                    JsonClaimHelper.CreateClaim(Resource1ClaimUri, new EdFiResourceClaimValue(ReadActionUri, "Override"))
+                    JsonClaimHelper.CreateClaim(Resource1ClaimUri, new EdFiResourceClaimValue(ReadActionUri, new List<string> { "Override" }))
```

Comment:
Convert to an array rather than creating lists.

Reply:
I noticed that you have updated it